uui/src/lib/Theme/theme.svelte

```diff
@@ -31,6 +37,26 @@
   // merge and set BreakPoint Context
   BREAKPOINT_CONTEXT.setBreakpoints({ ...UUI_BREAKPOINTS, ...breakpoints });
 
+  // Lifecycle
+  onMount(() => {
+    pageDocument = document;
+
+    // If preferredTheme is true, get theme from window and set in the theme store
+    // else set theme store to theme passed as prop
+    const themePreference = preferredTheme && getPreferredTheme(window);
+    if (themePreference) {
+      currentTheme.set(themePreference);
+    } else {
+      theme && currentTheme.set(theme);
```

Comment:
Is `theme &&` necessary since it's always defined?

Reply:
oh yeah it's not necessary. I'll remove that and update the docs.